android/app/build.gradle

```diff
@@ -313,22 +313,34 @@ dependencies {
     implementation 'com.facebook.fresco:animated-gif:2.0.0'
 }
 
-// Run this once to be able to run the application with BUCK
-// puts all compile dependencies into folder libs for BUCK to use
-
-task hemroidBuild(type: Exec) {
+def getLocalNDKDir = { ->
     def rootDir = project.rootDir
     def localProperties = new File(rootDir, "local.properties")
+    if (!localProperties.exists()) {
+        return null
+    }
+    Properties properties = new Properties()
+    localProperties.withInputStream { instr ->
+        properties.load(instr)
+    }
+    return properties.getProperty('ndk.dir')
+}
 
+// Run this once to be able to run the application with BUCK
+// puts all compile dependencies into folder libs for BUCK to use
+task hemroidBuild(type: Exec) {
```

Comment:
This is a ... curious ... function name.

Reply:
@rasom might know why it's called that based he added it in 8137c802fe7d39f7e4b82a8c34ec84e46198e7fb.